### ISSUE TITLE
Glue import with database and glue-table parameters returns all tables in a Glue database

### ISSUE THREAD

**Issue by roykoand** [datacontract/datacontract-cli#333]
Hi Team,

It's actually my first day using `datacontract-cli`, so I apologize in advance in case I am missing something...

I've tried the following command to export Glue table:
```
datacontract import --format glue --source X --glue-table Y
```
It returns all tables in X database instead of only Y. It does not seem to be expected per your documentation:
https://github.com/datacontract/datacontract-cli/blob/37cf8dc8711367010dd020379bee3920346f1507/README.md?plain=1#L940

Maybe there's something unique about my Glue database configuration? Let me know if you need any additional details. 

Thanks!

**Comment by jochenchrist** [comment]
[jverhoeks](https://github.com/jverhoeks) can you help here?

**Comment by roykoand** [comment]
There was a typo in `import_args.get("glue_tables")` which returned `table_names = None` instead of a `List[str]`

https://github.com/datacontract/datacontract-cli/pull/335

**Comment by jochenchrist** [comment]
fixed in #335
